docs/user/ppl/admin/catalog.rst

```diff
@@ -0,0 +1,88 @@
+.. highlight:: sh
+
+=================
+Catalog Settings
+=================
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+   :depth: 1
+
+Introduction
+============
+
+The concept of ``catalog`` is introduced to support the federation of SQL/PPL query engine to multiple data sources.
+This helps PPL users(plugins/customer) to leverage data from multiple data sources and derive correlation and insights.
+Catalog definition provides the information to connect to a datasource and also gives a name to them to refer in PPL commands.
+
+
+Definitions of catalog and connector
+====================================
+* Connector is a code component that adapts the query engine to a datasource. For example, Prometheus connector would adapt and help execute the queries to run on Prometheus data source.
```

Comment:
what is code component?  or just `Connector is a component that adapts the query engine to a datasource.`